index.html

```diff
@@ -58,81 +71,84 @@ <h2 class="h1">2023.10.16 (THU) ~ 17(FRI)</h2>
       <section class="section-container program-section">
         <h2 class="section-title">Main-program</h2>
```

Comment:
- [ ] I would highly recommend that you add background images to your design to make it more appealing like in the original design

![image](https://user-images.githubusercontent.com/98453979/210542973-6204b2c2-cb5d-4c3f-b66b-7ad450e9a1c0.png)


### Checkout original design

![image](https://user-images.githubusercontent.com/98453979/210543225-64662796-a08c-47e5-b8cd-d74ec41f43b9.png)